bench/bench.py

```diff
@@ -287,11 +289,11 @@ def decomm():
                            help='AWS account access key')
     tornado.options.define('secret_key', type=str, default='',
                            help='AWS account secret key')
-    tornado.options.define('ami', type=str, default='ami-48fd2120',
+    tornado.options.define('ami', type=str, default='ami-02df9ea15c1778c9c',
                            help='AMI ID')
     tornado.options.define('ssh_key_name', type=str, default='default',
                            help='SSH key name')
-    tornado.options.define('instance_type', type=str, default='c3.2xlarge',
+    tornado.options.define('instance_type', type=str, default='t2.medium',
```

Comment:
I think c4.xlarge or c5.xlarge would be more in the spirit of the previous default. A bit expensive, but they'll run for less than an hour, and t2 or t3 instances are probably not good for benchmarking because they are more oversubscribed and throttled.